doc/whatsnew/2.12.rst

```diff
@@ -3,11 +3,25 @@
 ***************************
 
 :Release: 2.12
-:Date: TBA
+:Date: 2021-11-24
 
 Summary -- Release highlights
 =============================
 
+In 2.12, we introduced a new option ``py-version`` that permits to analyse code for a python
+version that may differ from your current python interpreter. This does not affect all checkers but
+permits, for example, to check for python 3.5 code smells (using f-string's) while using pylint with python 3.6.
+The minimum version to run pylint is now 3.6.2, while the last working version for python 3.6.0
+and 3.6.1 was pylint 2.9.3.
+
+On top of fixing a lot of false positives and bugs, we also added new default checks, like
+``use-implicit-booleaness-not-comparison``, ``overridden-final-method``, and ``useless-with-lock``.
+There's also better check for TOML configurations.
+
+The checker for Yoda conditions is now an extension, you might want to enable it if you were
+relying on this check. There's also a new extension checker, ``consider-using-any-or-all`` that
+detects for loops that could be replaced by any or all, entirely contributed by @areveny,
+welcome to the team !
```

Comment:
We might want to add something about `end_line` here!

Reply:
Of course, it's one of the major feature (even if added ligning fast in the last week before release) ! Nice catch.